### PR TITLE
docs: Updating an existing booking

### PR DESCRIPTION
**Pull request by danjohnson95**
Closes designmynight/dmn#27488

### CODE REVIEW

File: site/content/api/booking-api/index.md [hunk 1]
```diff
@@ -234,6 +235,65 @@ status | The status of the booking: 'complete' for confirmed bookings; otherwise
 `last_name` | `string` the customer's last name
 `email` | `string` the customer's email address
 
+## Updating an existing booking
+
+An existing booking can be updated by making a `POST` request to the bookings API, providing you have the ID of the booking you wish to update. The following example will update the status of a booking to `in_progress`.
+
+```bash
+$ curl -X POST https://api.designmynight.com/v4/bookings/5b7e8070addee7612f456972
+  -d "status=in_progress"
```
Comment (StevePorter92) [inline site/content/api/booking-api/index.md]: does that need to be json?

File: site/content/api/booking-api/index.md [hunk 2]
```diff
@@ -234,6 +235,65 @@ status | The status of the booking: 'complete' for confirmed bookings; otherwise
 `last_name` | `string` the customer's last name
 `email` | `string` the customer's email address
 
+## Updating an existing booking
+
+An existing booking can be updated by making a `POST` request to the bookings API, providing you have the ID of the booking you wish to update. The following example will update the status of a booking to `in_progress`.
+
+```bash
+$ curl -X POST https://api.designmynight.com/v4/bookings/5b7e8070addee7612f456972
```
Comment (StevePorter92) [inline site/content/api/booking-api/index.md]: do you want to add a dummy auth header here too?

Comment (danjohnson95) [reply site/content/api/booking-api/index.md]: All the other examples in this file don't include the auth header. Do you reckon we should update all other examples to include one?

File: site/content/api/booking-api/index.md (hunk 1 above)
Comment (danjohnson95) [reply site/content/api/booking-api/index.md]: The data can be sent as json *or* as a plain URL encoded string like this. I've done the example like this as it's consistent with other examples, plus it saves us having to specify the `Content-Type: application/json` header

File: site/content/api/booking-api/index.md (hunk 2 above)
Comment (StevePorter92) [reply site/content/api/booking-api/index.md]: aa okay. I'll leave that up to you documentation @danjohnson95

File: site/content/api/booking-api/index.md (hunk 1 above)
Comment (StevePorter92) [reply site/content/api/booking-api/index.md]: +1 for consistency!

**Pull request merged**